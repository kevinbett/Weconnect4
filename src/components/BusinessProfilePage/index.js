import React from 'react';
import Navigation from '../Navigation/index';
import BusinessProfileBody from '../BusinessProfileBody/index';

const BusinessProfilePage = () => (

   <div>
       <Navigation />
       <BusinessProfileBody />
      
   </div>
);

export default BusinessProfilePage;