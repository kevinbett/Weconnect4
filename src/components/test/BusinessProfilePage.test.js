import BusinessProfilePage from "../BusinessProfilePage"
import { shallow } from "enzyme"

describe('Business Profile page',() => { 
    it ('renders Business Profile page', () => { 
        expect(shallow(<BusinessProfilePage/>)).toMatchSnapshot()
    });
} )