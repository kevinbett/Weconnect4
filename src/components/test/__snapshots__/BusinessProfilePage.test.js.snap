// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Business Profile page renders Business Profile page 1`] = `
<div>
  <Navigation />
  <withRouter(BusinessProfile) />
</div>
`;
